control_toolbox/include/control_toolbox/low_pass_filter.hpp

```diff
@@ -186,6 +188,35 @@ inline bool LowPassFilter<geometry_msgs::msg::WrenchStamped>::update(
   return true;
 }
 
+template <>
+inline bool LowPassFilter<std::vector<double>>::update(
+  const std::vector<double> & data_in, std::vector<double> & data_out)
+{
+  if (!configured_)
+  {
+    throw std::runtime_error("Filter is not configured");
+  }
+  // If this is the first call to update initialize the filter at the current state
+  // so that we dont apply an impulse to the data.
```

Comment:
Shouldn't the other filter's not initialize to zero in [configure()](https://github.com/ros-controls/control_toolbox/blob/ros2-master/control_toolbox/include/control_toolbox/low_pass_filter.hpp#L145-L151)? If I didn't do this I get an impulse applied to my feedback on startup which ends up causing my robot to move.

Reply:
Sounds reasonable. Do you want to fix that, maybe by configuring it with NaN and then set the appropriate initial value in the first update() call?